blog/views.py

```diff
@@ -82,3 +84,59 @@ def make_yaml(post, form):
     # write to YAML
     with open(r'./output_yaml/sample_output-' + str(post.id) + '.yaml', "w") as file:
         documents = yaml.dump(post_dict, file)
+
+
+def CLIP_form(request):
+    fastq = Fastq.objects.all()  # will list ALL fastq entries since the beginning of time
+
+    if request.method == 'POST':
+        form = CLIPForm(request.POST)
+        if request.POST.get("save"):
+            fastqs = []
+            for key in request.POST.keys():
+                try:
+                    if key.startswith('fqid_'):  # TODO: refactor, hacky
+                        Fastq.objects.get(pk=request.POST.get(key))
+                        fastqs.append(request.POST.get(key))
+                except Exception as e:
+                    print(e)  # TODO: log
+                    pass
+
+            if form.is_valid():
+                clip = form.save(commit=False)
+                clip.fastqs = ','.join(fastqs)  # fastqs is a CharField, save all fastq ids as str(comma-separated list)
+
+                clip.save()
+                CLIP_yaml(clip)
+                # set variables to field values
+                return redirect('/CLIP/')
```

Comment:
One thing I noticed is when we click the 'Add Item' button, all the previously filled field values disappear. I think after merging this PR, instead of redirecting it to an empty CLIP form we should redirect it to the filled out CLIP form. So everytime someone adds a fastq, they should not have to fill out the whole form again. I can add this as an issue